lib/constants/category.ts

```diff
@@ -1,24 +1,53 @@
-export type Category =
-  | 'ansible'
-  | 'batect'
-  | 'bazel'
-  | 'c'
-  | 'cd'
-  | 'ci'
-  | 'dart'
-  | 'docker'
-  | 'dotnet'
-  | 'elixir'
-  | 'golang'
-  | 'helm'
-  | 'iac'
-  | 'java'
-  | 'js'
-  | 'kubernetes'
-  | 'node'
-  | 'php'
-  | 'python'
-  | 'ruby'
-  | 'rust'
-  | 'swift'
-  | 'terraform';
+const Categories = [
+  'ansible',
+  'batect',
+  'bazel',
+  'c',
+  'cd',
+  'ci',
+  'dart',
+  'docker',
+  'dotnet',
+  'elixir',
+  'golang',
+  'helm',
+  'iac',
+  'java',
+  'js',
+  'kubernetes',
+  'node',
+  'php',
+  'python',
+  'ruby',
+  'rust',
+  'swift',
+  'terraform',
+] as const;
+
+export type Category = (typeof Categories)[number];
+
+export const CategoryNames: Record<Category, string> = {
```

Comment:
this should be part of the docs code, because it's totally unused by renovate.

it also seems to cause coverage issues.